### ISSUE TITLE
Table: Make table responsive for mobile

### ISSUE THREAD

**Issue by JoaoSilvaGrafana** [grafana/grafana#63946]
Currently none of our tables are easy to read on mobile, for example see https://github.com/grafana/grafana/issues/63797 .
There is a new table component `InteractiveTable` that we should use in admin-like pages, but that is not fully responsive either.
We need to figure out how to make tables easy to read on mobile, be it by making them scrollable or changing the design completely to maybe showing a list of cards.

**Comment by torkelo** [comment]
I think this is next to impossible, we should not use tables if we need responsive UIs

**Comment by Clarity-89** [comment]
> I think this is next to impossible, we should not use tables if we need responsive UIs 

I don't think this is an option, we have a lot of pages with tabular data in Admin and making them at leat usable should be possible.

I think the easiest would be to add a horizontal scroll to the InteractiveTable, it would at least take the component from broken to usable on mobile.

**Comment by torkelo** [comment]
true, adding scrolling could work but is a pretty bad mobile experience. Just saying if a good mobile experience is important we should not use table designs
